featuretools_tsfresh_primitives/primitives/fft_aggregated.py

```diff
@@ -30,3 +35,112 @@ def function(x):
             return list(fft_aggregated(x, param=param))[0][1]
 
         return function
+
+
+class ShortTermFftAggregated(TransformPrimitive):
```

Comment:
Curious, why is this called short term?

Reply:
It's the most widely used term. I don't see any literature that refers to it as a "rolling" Fourier. 
https://stackoverflow.com/questions/20771080/is-a-rolling-fft-possible-and-could-it-be-of-use
https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.stft.html
https://en.wikipedia.org/wiki/Short-time_Fourier_transform